src/ErrorProne.NET.Core/DiagnosticAnalyzerBase.cs

```diff
@@ -33,11 +33,10 @@ protected DiagnosticAnalyzerBase(params DiagnosticDescriptor[] diagnostics)
         public sealed override void Initialize(AnalysisContext context)
         {
             context.EnableConcurrentExecution();
-            
-            if (ReportDiagnosticsOnGeneratedCode)
-            {
-                context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.ReportDiagnostics);
-            }
```

Comment:
Note: Analyze | ReportDiagnostics is already the default. So, currently `ReportDiagnosticsOnGeneratedCode` is broken. This wasn't observed because there are actually no analyzers that overrides this property to return false. So, it was always true.